include/cute/arch/mma_xe.hpp

```diff
@@ -0,0 +1,56 @@
+
+#pragma once
+
+#include <cute/config.hpp>
+#include <cute/arch/mma.hpp>
+#include <cute/util/sycl_vec.hpp>
+
+#ifdef __SYCL_DEVICE_ONLY__ 
+#define SYCL_DEVICE_OCL(x) SYCL_EXTERNAL x
+#else 
+#define SYCL_DEVICE_OCL(x) inline x { assert(false); }
+#endif
```

Comment:
Same here, would it be possible to move this built-ins to include/cute/config.hpp file